src/pybind/mgr/dashboard_v2/controllers/dashboard.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from collections import defaultdict
+import collections
+
+import json
+import time
+
+import cherrypy
+
+from ..tools import ApiController, AuthRequired, BaseController
+
+
+@ApiController('dashboard')
+@AuthRequired()
+class Dashboard(BaseController):
+    @cherrypy.expose
+    @cherrypy.tools.json_out()
+    def toplevel(self):
+        fsmap = self.mgr.get("fs_map")
+        filesystems = [
+            {
+                "id": f['id'],
+                "name": f['mdsmap']['fs_name']
+            }
+            for f in fsmap['filesystems']
+        ]
+
+        return {
+            'health_status': self.health_data()['status'],
+            'filesystems': filesystems,
```

Comment:
Hm. exporting `filesystems` in `/api/dashboard/toplevel` seems strange to me.

Reply:
I will be using that information on the filesystem menu.